codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2HeadersDecoder.java

```diff
@@ -153,6 +184,15 @@ protected final boolean validateHeaders() {
         return validateHeaders;
     }
 
+    /**
+     * Determines if the header values should be validated as a result of the decode operation, in addition to the
+     * {@linkplain #validateHeaders() headers}.
+     * @return {@code true} if the header values should be validated as a result of the decode operation.
+     */
+    protected boolean validateHeaderValues() { // Not 'final' due to backwards compatibility.
```

Comment:
There could be sub-classes in the wild that define their own `validateHeaderValues`, so if we added a final method with the same signature, those classes would fail to compile.

Reply:
Agreed, this is an interesting case. On the other hand, if someone added a method with the same signature but lower visibility, they will be broken anyway.

Reply:
In these cases we follow what revapi does by default. Otherwise we won't have a way to expose this, which would be a broken symmetry.